src/main/java/igrad/logic/parser/CourseBookParser.java

```diff
@@ -114,6 +114,29 @@ public Command parseCommand(String userInput) throws ParseException, IOException
 
         switch (commandWord) {
 
+        /*
+         * If there is only one command word provided instead of the supposed two-word commands, flag an error
+         * to feedback to the user.
+         */
+
+        case CourseCommand.COURSE_COMMAND_WORD:
+            throw new ParseException(MESSAGE_UNKNOWN_COURSE_COMMAND);
+
+        case RequirementCommand.REQUIREMENT_COMMAND_WORD:
+            throw new ParseException(MESSAGE_UNKNOWN_REQUIREMENT_COMMAND);
+
+        case ModuleCommand.MODULE_COMMAND_WORD:
+            throw new ParseException(MESSAGE_UNKNOWN_MODULE_COMMAND);
+
+        case CourseAddCommand.COMMAND_WORD:
```

Comment:
but over here, you should add some comments or line breaks to 'demarcate` that from this point onwards, below are the actual commands